braintree/BTEncryption/BTAES.m

```diff
@@ -14,13 +14,13 @@ +(NSData*) encrypt:(NSData*) data withKey:(NSData*) key {
 +(NSData*) encrypt:(NSData *) data withKey:(NSData *) key Iv:(NSData *) iv {
   char keyPtr[kCCKeySizeAES256 + 1];
   bzero( keyPtr, sizeof( keyPtr ) );
-  [key getBytes:keyPtr];
+  [key getBytes:keyPtr length:sizeof(key)];
```

Comment:
@sebbean This could be a nit since I haven't worked with this code recently, but shouldn't we use `sizeof(keyPtr)` rather than `sizeof(key)`? Otherwise there is a risk of a buffer overrun when the `key` data is bigger than `keyPtr`.